src/components/UserLandingPage.vue

```diff
@@ -34,60 +34,237 @@
 
     <div class="row align-items-center justify-content-center text-center">
       <div class="col-lg-10 align-self-end">
-        <h1 class="text-black font-weight-bold">Welcome, ${Driver}</h1>
+        <hr class="divider my-4" />
+        <h1 class="text-black font-weight-bold">Welcome, {{user}}</h1>
         <hr class="divider my-4" />
       </div>
     </div>
 
-    <!-- ALL ORDERS -->
-    <div v-if="driverView === 'all'">
-      <b-col lg="4" class="pb-2">
-        <b-button size="lg" @click="$store.commit('sendSelectedOrders', driverSelectedOrders)">SuperDel Selected Order(s)</b-button>
-      </b-col>
-
-      <b-form-checkbox
-        v-for="item in items"
-        :key="item.key"
-        href="#"
-        class="flex-column align-items-start"
-        :checked="itemClick(item)"
-        variant="secondary"
-      >
-        <div class="d-flex w-100 justify-content-between">
-          <h5 class="mb-1">{{item.one}}</h5>
-          <small class="text-muted">{{item.two}}</small>
+
+<!-- USER Cart-->
+<div class="cart">
+  <b-list-group>
+    <b-card bg-variant="dark" text-variant="white" title="Checkout">
+      <b-card-text>
+        Thank you for shopping with us, {{user}}!
+        <b-card-text class="text-muted" >Distincts: {{cartItems.length}}</b-card-text>
+        <div class="float-right">
+        <b-button href="#" variant="primary" v-on:click="checkout">Checkout</b-button>
```

Comment:
Shortcut v-on:click as `@click`. This is more for consistency since we've been using that elsewhere.